src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ArtistsFilter.tsx

```diff
@@ -96,8 +96,12 @@ export const ArtistsFilter: FC<ArtistsFilterProps> = ({
     filterContext.currentlySelectedFilters()["includeArtworksByFollowedArtists"]
   const followedArtistArtworkCount = filterContext?.counts?.followedArtists ?? 0
 
+  const selection = filterContext.currentlySelectedFilters().artistIDs
+  const hasSelection =
+    (selection && selection.length > 0) || isFollowedArtistCheckboxSelected
+
   return (
-    <FilterExpandable label="Artists" expanded={expanded}>
+    <FilterExpandable label="Artists" expanded={hasSelection || expanded}>
```

Comment:
The time period and color filters were not being collapsed-on-select because they were able to override the `expanded` prop if a value was selected. (They would, however, collapse when the values were unselected.) Therefore, I went and added the same behavior to all filters.

Reply:
yup, makes sense.